nuxeo-web-ui-ftest/test/pages/ui/browser/document_versions.js

```diff
@@ -72,7 +72,7 @@ export default class DocumentVersions extends BasePage {
     const itemClicked = items.some((item) => {
       const foundLabel = item.getText('.title').trim();
       if (foundLabel === label) {
-        item.click();
+        item.elements('.title').click();
```

Comment:
I'd even put a `this.list.waitForVisible('div[name="version-item"] .title')` at the beginning of this method, to make sure we're waiting for at list a version to be there.